app/jobs/replication/delivery_dispatcher_job.rb

```diff
@@ -18,7 +18,7 @@ module Replication
   #   the VM is updated to a new vesrion). Therefore, we receive the zip
   #   metadata from the process that actually created the zip file.
   class DeliveryDispatcherJob < Replication::ZipPartJobBase
-    queue_as :zips_made
+    queue_as :replication_plexer
```

Comment:
Can we call `delivery dispatcher`? We're deprecating "plexer".

Reply:
yeah - that was from the ticket which was written before this class changed.  So I agree with JLitt's request:  `replication_delivery_dispatcher`.

Reply:
How do you feel about that suggestion @ndushay? I made these based on the ticket.

Reply:
I wrote the ticket and it was out of date.  Sorry about that.